README.md

```diff
@@ -1,73 +1,29 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+#
+# mobile Version
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+To launch the project you need a modern browser and double click on **index.html**
```

Comment:
- [ Optional ] It is good to have the subsection under the Getting Started section for readability and it helps make your README look professional. Assume, I am visiting your repo and am a complete novice to HTM, CSS, Linters, etc; can you give me the steps I need to follow to be able to replicate your project locally?